apps/prairielearn/src/lib/db-types.ts

```diff
@@ -823,6 +841,7 @@ export const QuestionSchema = z.object({
   external_grading_timeout: z.number().nullable(),
   grading_method: z.enum(['Internal', 'External', 'Manual']),
   id: IdSchema,
+  is_draft: z.boolean(),
```

Comment:
I checked our other boolean columns and we don't have an `is_` prefix elsewhere. I'm wondering if we should just call this `draft` instead? cc @mwest1066